CONTRIBUTING.md

```diff
@@ -36,7 +36,7 @@ and docs help a lot with that.
 - missing tests : Encourage people to do TDD by making clear a PR will not get merged
 if it lacks tests. Not everything is easy to test though, keep that in mind.
 - BC breaks : If there is a BC-break, ask for a deprecation system to be created instead,
-and make sure the `master` branch is used.
+and make sure the unstable branch is used.
```

Comment:
We know the unstable branch pet project and could add it here to be more specific instead of talking about "unstable"

Reply:
But this is the global contributing guide , not the project specific